tests/_support/rest-api-tester/assets/js/src/trap-script.ts

```diff
@@ -0,0 +1,180 @@
+(
```

Comment:
I've used Typescript here just for convenience (I *really* like strong typehinting) but it is just a source tool. PHP storm supports it out of the box and it's very similar to PHP as syntax.